### ISSUE TITLE
Oil hole

### ISSUE THREAD

**Issue by navaroneee** [pWn3d1337/Techguns#116]
Have a special line on JSON (COFH) to spawn oil holes? can't do its works

**Comment by Krid3l** [comment]
I don't think so. Buildcraft, however, spawns oil geysers and pools in rivers and oceans.

**Comment by navaroneee** [comment]
Nevermind, I solved, if i waiting from reply...haha
